src/components/Parts/Editor/EditorList.tsx

```diff
@@ -9,6 +9,7 @@ import HeadingBox from './Internals/HeadingBox'
 export interface EditorListProps {
   kind: string
   query?: string
+  count?: number
```

Comment:
@shashigharti 
Can we just use ScrollBox inside EditorList? Removing count and ScrollBox logic in the component users?

Reply:
We probably need to pass "height" then

Reply:
BTW it also has broken SchemaEditor because it doesn't pass `count`. Nothing serious I just fixed it but we need to be very cautious about changes like this